.gitattributes

```diff
@@ -1,5 +1,3 @@
-boilerplate/** linguist-generated=true
```

Comment:
Why is this being deleted?

Reply:
I think this was a casualty as part of the SDKv1 migration process which involves re-onboarding the operator into a fresh boilerplate. But, should not have been dropped intentionally - I've pushed a commit which restores this configuration.